collectors/python.d.plugin/uwsgi/uwsgi.chart.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+# Description: uwsgi netdata python.d module
+# Author: Robbert Segeren (robbert-ef)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+from copy import deepcopy
+from bases.FrameworkServices.SocketService import SocketService
+
+# default module values (can be overridden per job in `config`)
+# update_every = 2
+priority = 60000
+retries = 60
+
+ORDER = ['requests', 'tx', 'avg_rt', 'memory_rss', 'memory_vsz', 'exceptions', 'harakiri', 'respawn', ]
+
+# NOTE: lines are created dynamically in `check()` method
+CHARTS = {
+    'requests': {
+        'options': [None, 'Requests', 'requests/s', 'requests', 'uwsgi.requests', 'stacked'],
+        'lines': [
+            ['requests.%s', 'worker %s', 'incremental']
+        ]},
+    'tx': {
+        'options': [None, 'Transmitted data', 'KB/s', 'requests', 'uwsgi.tx', 'stacked'],
+        'lines': [
+            ['tx.%s', 'worker %s', 'incremental']
+        ]},
+    'avg_rt': {
+        'options': [None, 'Average request time', 'ms', 'requests', 'uwsgi.avg_rt', 'line'],
+        'lines': [
+            ['avg_rt.%s', 'worker %s', 'absolute']
+        ]},
+    'memory_rss': {
+        'options': [None, 'RSS (Resident Set Size)', 'MB', 'memory', 'uwsgi.memory_rss', 'stacked'],
+        'lines': [
+            ['memory_rss.%s', 'worker %s', 'absolute', 1, 1024 * 1024]
+        ]},
+    'memory_vsz': {
+        'options': [None, 'VSZ (Virtual Memory Size)', 'MB', 'memory', 'uwsgi.memory_vsz', 'stacked'],
+        'lines': [
+            ['memory_vsz.%s', 'worker %s', 'absolute', 1, 1024 * 1024]
+        ]},
+    'exceptions': {
+        'options': [None, 'Exceptions', 'exceptions', 'exceptions', 'uwsgi.exceptions', 'line'],
+        'lines': [
+            ['exceptions.%s', 'worker %s', 'incremental']
+        ]},
+    'harakiri': {
+        'options': [None, 'Harakiris', 'harakiris', 'harakiris', 'uwsgi.harakiris', 'line'],
+        'lines': [
+            ['harakiri_count.%s', 'worker %s', 'incremental']
+        ]},
+    'respawn': {
+        'options': [None, 'Respawns', 'respawns', 'respawns', 'uwsgi.respawns', 'line'],
+        'lines': [
+            ['respawn_count.%s', 'worker %s', 'incremental']
+        ]},
+}
+
+
+class Service(SocketService):
+    def __init__(self, configuration=None, name=None):
+        SocketService.__init__(self, configuration=configuration, name=name)
```

Comment:
Please change this to:
```
super().__init__(configuration=configuration, name=name)
```

It does the same thing in a more future-proof manner and is more consistent with other modules.